File: Grupo 2-3/C Sharp/funciones/Ejercicios/monedas.cs
using System;

class Principal
{

static int CambiarMonedas(ref int cantidad, int tipo){
   int num;
   num = cantidad / tipo;
   cantidad = cantidad % tipo;
   return num;
}
static void Main(){

   
   int dinero = int.Parse(Console.ReadLine());

   Console.WriteLine("De 500 hay {0}", CambiarMonedas(ref dinero, 500)); 
   Console.WriteLine("De 200 hay {0}", CambiarMonedas(ref dinero, 200)); 
   Console.WriteLine("De 100 hay {0}", CambiarMonedas(ref dinero, 100)); 
   Console.WriteLine("De 50  hay {0}", CambiarMonedas(ref dinero, 50)); 
   Console.WriteLine("De 25  hay {0}", CambiarMonedas(ref dinero, 25)); 
   Console.WriteLine("De 10  hay {0}", CambiarMonedas(ref dinero, 10)); 
   Console.WriteLine("De 5   hay {0}", CambiarMonedas(ref dinero, 5)); 
   Console.WriteLine("De 1 hay {0}", CambiarMonedas(ref dinero, 1)); 

}

}

File: Grupo 2-1/Lenguaje C/funciones/ Ejercicios/monedas.c
#include <stdio.h>

int cantidadBilletes(int *dinero, int tipo){

    int num = (*dinero) / tipo;
    (*dinero) = (*dinero) % tipo;
    return num;
}

int main()
{
   int cantidad;
    scanf ("%d", &cantidad);
    printf("De 500 hay %d\n",cantidadBilletes(&cantidad, 500));
    printf("De 200 hay %d\n", cantidadBilletes (&cantidad , 200));
    printf("De 100 hay %d\n", cantidadBilletes (&cantidad , 100));
    printf("De 50  hay %d\n", cantidadBilletes (&cantidad , 50));
    printf("De 25  hay %d\n", cantidadBilletes (&cantidad , 25));
    printf("De 10  hay %d\n", cantidadBilletes (&cantidad , 10));
    printf("De 5   hay %d\n", cantidadBilletes (&cantidad , 5));
    printf("De 1   hay %d\n", cantidadBilletes (&cantidad , 1));
    
    





    return 0;
}
